owncloudApp/src/main/res/layout/list_fragment.xml

```diff
@@ -18,80 +18,117 @@
-            android:layout_height="match_parent"
-            android:footerDividersEnabled="false"
-            android:visibility="gone">
+            android:layout_height="0dp"
+            android:layout_marginTop="4dp"
+            fab:layout_constraintBottom_toBottomOf="parent"
+            fab:layout_constraintTop_toBottomOf="@+id/options_layout">
 
-            <third_parties.in.srain.cube.GridViewWithHeaderAndFooter
-                android:id="@+id/grid_root"
+            <androidx.swiperefreshlayout.widget.SwipeRefreshLayout
+                android:id="@+id/swipe_containing_list"
                 android:layout_width="match_parent"
                 android:layout_height="match_parent"
-                android:columnWidth="100dp"
-                android:gravity="center"
-                android:horizontalSpacing="2dp"
-                android:stretchMode="columnWidth"
-                android:verticalSpacing="2dp"
-                android:visibility="visible" />
-        </androidx.swiperefreshlayout.widget.SwipeRefreshLayout>
-
-        <androidx.swiperefreshlayout.widget.SwipeRefreshLayout
-            android:id="@+id/swipe_containing_empty"
-            android:layout_width="match_parent"
-            android:layout_height="match_parent"
-            android:visibility="gone">
+                android:footerDividersEnabled="false"
+                android:visibility="visible">
 
-            <ScrollView
+                <com.owncloud.android.ui.ExtendedListView
+                    android:id="@+id/list_root"
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent"
+                    android:divider="@color/list_divider_background"
+                    android:dividerHeight="1px"
+                    android:visibility="visible"
+                    tools:listitem="@layout/item_file_list" />
+            </androidx.swiperefreshlayout.widget.SwipeRefreshLayout>
+
+            <androidx.swiperefreshlayout.widget.SwipeRefreshLayout
+                android:id="@+id/swipe_containing_grid"
                 android:layout_width="match_parent"
-                android:layout_height="match_parent">
+                android:layout_height="match_parent"
+                android:footerDividersEnabled="false"
+                android:visibility="gone">
 
-                <TextView
-                    android:id="@+id/empty_list_view"
+                <third_parties.in.srain.cube.GridViewWithHeaderAndFooter
+                    android:id="@+id/grid_root"
                     android:layout_width="match_parent"
-                    android:layout_height="wrap_content"
-                    android:layout_gravity="center"
-                    android:gravity="center_vertical|center_horizontal"
-                    android:text="@string/empty"
+                    android:layout_height="match_parent"
+                    android:columnWidth="100dp"
```

Comment:
Hardcoded dimen, you have other ones in this xml

Reply:
We will take care of it in the close future. We need to refactor this layout in upcoming redesign issues 👍